experimental/oak_async/tests/tests.rs

```diff
@@ -285,6 +288,63 @@ fn many_reads_parallel() {
     assert_eq!(data.len(), 1000);
 }
 
+// Stream functionality relies heavily on the implementation of `Future`, so we only need minimal
+// tests for the `Stream` impl.
+#[test]
+fn stream_immediate() {
```

Comment:
Perhaps give this a more descriptive name if possible.

Reply:
Done.